pkg/operator/ceph/pool/controller.go

```diff
@@ -288,3 +311,32 @@ func updateStatus(client client.Client, poolName types.NamespacedName, status st
 	}
 	logger.Debugf("pool %q status updated to %q", poolName, status)
 }
+
+func configureRBDStats(clusterContext *clusterd.Context, clusterName string) error {
+	cephBlockPoolList := cephv1.CephBlockPoolList{}
+	var enableStatsForPools []string
+	err := clusterContext.Client.List(context.TODO(), &cephBlockPoolList)
+	if err != nil {
+		return errors.Wrapf(err, "failed to retrieve list of CephBlockPool")
+	}
+	for _, cephBlockPool := range cephBlockPoolList.Items {
+		if cephBlockPool.GetDeletionTimestamp() == nil && cephBlockPool.Spec.EnableRBDStats {
+			// list of CephBlockPool with enableRBDStats set to true and not marked for deletion
+			enableStatsForPools = append(enableStatsForPools, fmt.Sprintf("%s/%s", cephBlockPool.Name, cephBlockPool.Namespace))
```

Comment:
Why is the namespace included? Ceph doesn't know anything about the k8s namespace of the pool. Isn't this just a list of raw pool names?

Reply:
According to the documentation, its `poolName/Namespace`. if namespace is not provided, `poolName` will be picked from all namespaces. So, I assumed it'd be same as k8s namespace, how else would that name be decided?

Reply:
let's not confuse kubernetes namespace and ceph namespace, the doc refers to ceph pool namespace so we don't need `cephBlockPool.Namespace`

Reply:
oh I did suspect that was the case. Will fix it.